aten/src/ATen/native/native_functions.yaml

```diff
@@ -507,6 +507,12 @@
 
 - func: bilinear(Tensor input1, Tensor input2, Tensor weight, Tensor? bias) -> Tensor
 
+- func: binary_cross_entropy(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean) -> Tensor
+  variants: function
```

Comment:
these should both have `python_module: nn` and you need to keep the `out` variants as well.